lib/cpp/build/bin/clang++

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env sh
```

Comment:
Let's remove this indirection and just inline it in the two places it's used to keep it simple.